apps/etl-func/src/adapters/pdv-tokenizer/pdv-tokenizer-client.ts

```diff
@@ -44,13 +44,8 @@ export default class PDVTokenizerClient implements TokenizerClient {
 
       assert.strictEqual(
         response.ok,
-        false,
-        new Error(
-          `Error in tokenizer api call with status ${response.status}`,
-          {
-            cause: problemSchema.parse(responseJson),
-          },
```

Comment:
why did you remove this? The cause contains the error response of the PDV api and it could be useful to debug erros

Reply:
I was having a problem running the pdv tokenization part. It seems that problemSchema.parse(responseJson) executes whether the assertion is successfull or not. So if the assertion is successfully(it means the tokenize api call goes well) it trhows an error because responseJson doesn't have the expected schema.

Reply:
that's very strange, that part should only run if the error is thrown. If this is the case I think if will be better to have an if instead of the assertion in order to put the error response in the cause